_posts/2023-01-02-release.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Release 2023.01
+author: priya
+date: 2023-01-02
+category: Jekyll
+layout: post
+---
+
+### Defect fixes
+
+* Column browser was enabled when there was a pipeline error.
+* Pipeline in error -Tooltip message was not showing up on hover for disabled items.
+* Appropriate error message was not appearing when user tries to delete foreign view.
+* Data type was not visible for long column name in report.
```

Comment:
When the column name is long, the data type is not visible in the Report. This has been fixed